spec/lib/orbf/rules_engine/data/contract_validation_exception_spec.rb

```diff
@@ -0,0 +1,32 @@
+RSpec.describe Orbf::RulesEngine::ContractValidationException do
+  it "has custom messages on missing org unit attributes" do
+    missing_attrs = ["contract_start_date"]
+    org_unit = { "id" => "cLc2uthCRfm", "name" => nil, "path" => "/pL5A7C1at1M/BmKjwqc6BEw/DkK8tVZ6xfJ/z6sJvc6NR59/cLc2uthCRfm" }
+    exception = Orbf::RulesEngine::ContractValidationException.new(missing_attrs, org_unit)
+    
+    expect(exception.message).to match(/#{"name - missing"}/)
+  end
+
+  it "has custom messages on missing org unit" do
+    missing_attrs = ["contract_start_date"]
+    exception = Orbf::RulesEngine::ContractValidationException.new(missing_attrs, org_unit=nil)
```

Comment:
I have never thought of doing a `org_unit=nil` as a paremeter to be passed, makes sense that it works :)